Server/ConcordiaCurriculumManager/Controllers/CourseController.cs

```diff
@@ -96,4 +96,38 @@ public async Task<ActionResult> InitiateCourseCreation([FromBody, Required] Cour
                 statusCode: StatusCodes.Status500InternalServerError);
         }
     }
+
+    [HttpPut(nameof(InitiateCourseModification))]
+    [Consumes("application/json")]
```

Comment:
Can you change this to `[Consumes(typeof(CourseModificationInitiationDTO), MediaTypeNames.Application.Json)]` (and do likewise for the course creation initiation endpoint?). This is the pattern in the AuthenticationController https://github.com/JamesPartsafas/ConcordiaCurriculumManager/blob/b4aefb9d14f881d6a4e42588a5d5514d517b839e/Server/ConcordiaCurriculumManager/Controllers/AuthenticationController.cs